tests/Rule/LatteTemplatesRule/CustomLatteTemplateResolver/Fixtures/TestingCustomTemplateResolver.php

```diff
@@ -26,7 +25,7 @@ public function collect(): array
         ];
     }
 
-    protected function getResult(CollectedResolvedNode $resolvedNode, LatteContextData $latteContext): LatteTemplateResolverResult
+    public function resolve(CollectedResolvedNode $resolvedNode, LatteContext $latteContext): LatteTemplateResolverResult
     {
         $result = new LatteTemplateResolverResult();
         $result->addTemplate(new Template(
```

Comment:
What is the difference between addTemplate and other methods? why we have some many options? I'm author / co-author of this package and I don't understand it myself :D When should I use what? Ideal scenario - always use the same template

Reply:
$resolvedNode->getParam() returns string, I have multi-template controls and I have to send array to these params

Reply:
Ok we can allow array too

Reply:
Othet methods are just helpers for most common task. All use addTemplate internally.

Reply:
As I look at it it would probably need some cleanup. Any idea how to restructure it?

Reply:
FromRenders do some additional checks that shoukd be moved to FromRender. And then FromRenders coukd be probably removed and foreach moved to resolver

Reply:
ok